components/post_view/post_flag_icon/post_flag_icon.js

```diff
@@ -84,45 +82,47 @@ export default class PostFlagIcon extends React.PureComponent {
     }
 
     render() {
-        if (this.props.isEphemeral) {
-            return null;
-        }
-
         const isFlagged = this.props.isFlagged;
 
         const flagVisible = isFlagged ? 'visible' : '';
 
+        const iconClass = 'icon icon--small';
+
         let flagIcon;
         if (isFlagged) {
-            flagIcon = <FlagIconFilled className='icon'/>;
+            flagIcon = <FlagIconFilled className={iconClass + ' icon--small-filled'}/>;
```

Comment:
3/5 prefer the use of 'classNames'

Reply:
Done, but still needed to leave the `if` `else` since we are returning different elements. No sure if classnames actually provides that much value here this way. But it there now anyway 🙂